ingestion_server/ingestion_server/api.py

```diff
@@ -87,17 +87,23 @@ def on_post(self, req, resp):
         # Inject shared memory
         progress = Value("d", 0.0)
         finish_time = Value("d", 0.0)
+        active_workers = Value(
+            "i", int(False)
+        )  # Tracks whether the task has any active distributed workers
```

Comment:
If you move this comment to the line above, `black` won't try to line break it:
```suggestion
        # Tracks whether the task has any active distributed workers
        active_workers = Value("i", int(False))
```

Reply:
Visually, I felt like moving the comment inline makes it clear that `active_workers` is part of the group of values including `progress` and `finish_time` that are underneath the `Inject shared memory` comment, without 'breaking up' that block of code:

```
        # Inject shared memory
        progress = Value("d", 0.0)
        finish_time = Value("d", 0.0)
        active_workers = Value(
            "i", int(False)
        )  # Tracks whether the task has any active distributed workers
```

Maybe overthinking and I don't feel strongly about it 😄

Reply:
Ha, that's fair! I've corrected a lot of these cases in the past where it wasn't necessary, but I agree - here it makes sense to keep it the way you have it :slightly_smiling_face: